cpp/src/command_classes/CentralScene.cpp

```diff
@@ -80,7 +99,7 @@ bool CentralScene::RequestState
 {
 	Log::Write(LogLevel_Info, GetNodeId(), "CentralScene RequestState: %d", _requestFlags);
 	bool requests = false;
-	if( ( _requestFlags & RequestFlag_AfterMark ))
+	if( ( _requestFlags & RequestFlag_Dynamic ))
```

Comment:
Thanks,  also added `SetStaticRequest( StaticRequest_Values );` in the contructor or else the `HasStaticRequest( StaticRequest_Values )` didn't return true. Is this the right way?

```
CentralScene::CentralScene
(
		uint32 const _homeId,
		uint8 const _nodeId
):
CommandClass( _homeId, _nodeId ),
m_scenecount(0)
{
    SetStaticRequest( StaticRequest_Values );
	Log::Write(LogLevel_Info, GetNodeId(), "CentralScene - Created %d", HasStaticRequest( StaticRequest_Values ));
}
```